lib/parallel_tests/test/runner.rb

```diff
@@ -248,7 +248,7 @@ def find_tests(tests, options = {})
             else
               file_or_folder
             end
-          end.uniq
+          end.uniq.sort
```

Comment:
Some users might expect us to honor the order they passed things in, so `files_in_folder` should sort, but the rest should not ?

Reply:
This makes sense. I am removing the `.sort` from the `find_tests`. The `files_in_folder` already sorts, so no need for me to do anything there.